src/System.Runtime.WindowsRuntime/src/System/IO/BufferedStreamWrapper.CoreRT.cs

```diff
@@ -0,0 +1,208 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.WindowsRuntime.Internal;
+using System.Threading;
+using System.Collections.ObjectModel;
+using System.Security;
+using System.Threading.Tasks;
+
+namespace System.IO
+{
+    /// <summary>
+    /// A BufferedStream wrapper which expose BufferedStream's UnderlyingStream and BufferSize
```

Comment:
In CoreCLR, BufferedStream is defined in System.IO and there are two internal properties which contains [FriendAccessAllowed] and WindowsRuntimeStreamExtensions can access these two internal fields.
In latest CoreRT(UAP101AOT), BufferedStream is also defined in System.IO and these two properties are still internal. Unless Friend Assembly is supported in CoreRT, WindowsRuntimeStreamExtensions can't access these two internal properties.

The wrapper is just to provide a way to access these two internal properties.

see CoreCLR definitions:
https://github.com/dotnet/coreclr/blob/85ff1e3b3ca3d7eb10ab0e2e6a41bdf721be4540/src/mscorlib/src/System/IO/BufferedStream.cs#L190
https://github.com/dotnet/coreclr/blob/85ff1e3b3ca3d7eb10ab0e2e6a41bdf721be4540/src/mscorlib/src/System/IO/BufferedStream.cs#L197